test/hummingbot/connector/derivative/dydx_perpetual/test_dydx_perpetual_derivative.py

```diff
@@ -266,55 +278,84 @@ def test_update_funding_fails_on_general_exception(self, get_markets_mock: Async
 
         self.check_is_logged(log_level="NETWORK", message="Unknown error.")
 
-    def test_tick_manual_poll_interval(self):
-        poll_interval = 10
-        exchange = DydxPerpetualDerivative(
-            dydx_perpetual_api_key="someAPIKey",
-            dydx_perpetual_api_secret="someAPISecret",
-            dydx_perpetual_passphrase="somePassPhrase",
-            dydx_perpetual_account_number=1234,
-            dydx_perpetual_ethereum_address="someETHAddress",
-            dydx_perpetual_stark_private_key="1234",
-            trading_pairs=[self.trading_pair],
-            poll_interval=poll_interval,
-        )
+    def test_tick_initial_tick_successful(self):
+        start_ts: float = time.time() * 1e3
 
-        initial_timestamp = 0
-        exchange._last_timestamp = initial_timestamp
-        exchange.tick(timestamp=initial_timestamp + poll_interval - 1)
+        self.exchange.tick(start_ts)
+        self.assertEqual(start_ts, self.exchange._last_poll_timestamp)
+        self.assertTrue(self.exchange._poll_notifier.is_set())
 
-        self.assertFalse(exchange._poll_notifier.is_set())
+    @patch("time.time")
```

Comment:
It might be problematic to patch something as basic and and low level as time.time. We shall consider doing. the same for it that we doo for sleep function